src/index.js

```diff
@@ -6,24 +6,78 @@ import './css/base.scss';
 
 //JS FILES
 import Traveler from './traveler.js';
+import Trip from './trip.js';
 import fetchAPI from './fetch.js';
+import domUpdates from './domUpdates.js';
 
 //QUERY SELECTORS
+const bookAFlightBtn = document.querySelector('.book-now-btn');
+const claculateNewTripCostBtn = document.querySelector('.calculate-cost-button');
+const returnToDashboardBtn = document.querySelector('.return-to-dashboard');
+const submitTripBtn = document.querySelector('.submit-request');
+const formDestinationInput = document.querySelector('.destinations-list');
+const formDurationInput = document.querySelector('.select-duration');
+const formTravelersInput = document.querySelector('.select-travelers');
+const formDateInput = document.querySelector('.select-date');
 
 //EVENT LISTENERS
 window.addEventListener('load', generateUsersInfo);
+bookAFlightBtn.addEventListener('click', function() {
+  domUpdates.displayBookFlightForm(destinationsData);
+});
+claculateNewTripCostBtn.addEventListener('click', claculateNewTripCost);
+returnToDashboardBtn.addEventListener('click', domUpdates.returnToDashboard);
+submitTripBtn.addEventListener('click', addTripRequest);
 
 //GLOBAL VARIABLES
 let currentUser;
 let destinationsData;
+let tripsData;
 
-//FETCH DATA
+//FETCH DAT
 function generateUsersInfo() {
-  Promise.all([fetchAPI.fetchUserData(1), fetchAPI.fetchTripsData(), fetchAPI.fetchDestinationsData()])
+  Promise.all([fetchAPI.fetchUserData(6), fetchAPI.fetchTripsData(), fetchAPI.fetchDestinationsData()])
     .then(data => {
       currentUser = new Traveler(data[0]);
+      tripsData = data[1];
       destinationsData = data[2];
-      currentUser.findUsersTrips(data[1].trips, destinationsData);
+      currentUser.findUsersTrips(tripsData.trips, destinationsData);
       currentUser.calculateTotalSpent();
+      domUpdates.generateWelcomeBanner(currentUser.getUsersFirstName());
+      domUpdates.placeCardsInCorrectSection(currentUser.trips, destinationsData);
     })
 }
+
+function claculateNewTripCost() {
+  let newTrip = {
+    destinationID: `${formDestinationInput.value}`,
+    duration: `${formDurationInput.value}`,
+    travelers: `${formTravelersInput.value}`,
+    cost: 0
+  };
+  let instantiateNewTrip = new Trip(newTrip);
+  instantiateNewTrip.calculateTotalCostofTrip(destinationsData);
+  domUpdates.displayNewTripCost(instantiateNewTrip);
+}
+
+function addTripRequest() {
+  let convertDate = new Date(formDateInput.value).toLocaleDateString('en-ZA');
+  let newTrip = {
+    id: tripsData.trips.length + 1,
```

Comment:
I ran into an issue with this idea because I wasn't updating my trips after posting so all the IDs were the same and I got POST errors. Looks like you are doing that already. This is great stuff man.